releases/kovan/active

```diff
@@ -1 +1 @@
-1.1.2
\ No newline at end of file
+1.1.3
```

Comment:
Didn't we agree that the active link is updated only after the spell is cast?

Reply:
Ah, that's kovan. All good then.

Reply:
For Kovan though?

Reply:
Ah, I just updated the symlink for completeness. I am pretty sure I updated mainnet also. i can undo it, but we'll need to remember to update this once it's cast.